docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2129,15 +2129,15 @@ The following error message codes may appear on the app-server log. Use the foll
 
 ### ZWED0087W
 
-  Tomcat PID=_pid_: stderr=_data_
+  Tomcat PID=_pid_: stderr=_error_
 
   **Reason:**
 
-  While stopping Tomcat, Tomcat process with Tomcat _pid_ encountered an standard error.
+  While stopping Tomcat, Tomcat process with Tomcat _pid_ encountered an _error_ (stderr).
 
   **Action:**
 
-  Action depends on what _data_ is and may be useful to debug.
+  Action depends on what _error_ is and may be useful to debug.
```

Comment:
Review the _error_ and if app server service is interrupted, send it and your log to support for troubleshooting.